opendm/types.py

```diff
@@ -478,7 +479,7 @@ def __init__(self, root_path, images_path, gcp_file = None):
         self.odm_georeferencing_las_json = io.join_paths(
             self.odm_georeferencing, 'las.json')
         self.odm_georeferencing_model_las = io.join_paths(
```

Comment:
Should we rename this variable to `odm_georeferencing_model_laz`?

Reply:
For consistency, I think so.

Reply:
I think @pierotofy  was talking about `odm_georeferencing_model_las`.